src/api/news/queries.ts

```diff
@@ -2,18 +2,20 @@ import { useQuery } from '@tanstack/react-query';
 import { Sections } from '@/constants/routes';
 import { fetchArticleByUri, fetchArticles } from './module';
 
-enum QueryKeys {
+export enum QueryKeys {
```

Comment:
do not export this enum.

Create method here like `prefetchArticles`

Reply:
Done. Created prefetchArticles() and prefetchSingleArticle() methods